gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -355,6 +355,16 @@ def __init__(self, controller, config):
         self.config_driver = heat_driver.HeatDriver(config)
         neutron_context = n_context.get_admin_context()
         self.configurator_rpc = NSOConfiguratorRpcApi(neutron_context, config)
+        self.status_map = {
+                'pt_add': {'status': 'PT_ADD_IN_PROGRESS',
```

Comment:
Currently, ACTIVE/ERROR/PENDING_CREATE/PENDING_DELETE/PENDINGUPDATE are the possible status values and they are defined in constants file. Can these also go there so that we know all possible status values?